README.md

```diff
@@ -1 +1,41 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse App
+
+> This is a simple Microverse task that demostrates the importance of using Linters
+
+## Built With
+
+- Html,Css,
+- No Frameworks
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+All you need is a compiler
+
+### Setup
+
+You will need to clone the project
```

Comment:
[OPTIONAL] I think it would be better if you told us how to clone your project. You can see an example in this repository. You can also check out other ideas from it

Reply:
@fobadara Thank you for the feedback . I will apply the comments / corrections accordingly. Cheers!!!